### PR TITLE
Static LMP & DOB options for test MCTS data

### PR DESCRIPTION
**Pull request by frankhuster**
Calling
```
python ~/src/mim/performance/data/data.py --cleardb --mctsmoms=10000 --lmp
```
with the `--lmp` parameter will generate mothers with a static LMP of 90 days prior to today.
And calling 
```
python ~/src/mim/performance/data/data.py --cleardb --mctskids=10000 --dob
```
with the `--dob` parameter will generate kids with a static DOB of today

### CODE REVIEW

File: testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java [hunk 1]
```diff
@@ -62,15 +62,21 @@ public String createSubscriptionPacks() {
 
     @RequestMapping(value = "/createMctsMoms")
     @ResponseBody
-    public String createMctsMoms(@RequestParam(required = true) int count) throws IOException {
-        return testingService.createMctsMoms(count);
+    public String createMctsMoms(
+            @RequestParam(required = true) int count,
```
Comment (koshalt) [inline testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java]: might be a silly question but how does this work for optional params?

Comment (frankhuster) [inline testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java]: this param isn't optional 
but the --lmp argument to data.py is
:-)

Comment (koshalt) [inline testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java]: right, and looking at data.py from above - 
`if args.mctsmoms > 0:
-        exec_http_get("{}/module/testing/createMctsMoms".format(args.server), {'count': args.mctsmoms})		+        params = {'count': args.mctsmoms, 'lmp': args.lmp > 0}
+        exec_http_get("{}/module/testing/createMctsMoms".format(args.server), params)`

How does this work? What kind of trickery is this? :confounded:

Comment (frankhuster) [inline testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java]: kosh == :cold_sweat: right now, but after reading [this](http://stackoverflow.com/questions/394809/does-python-have-a-ternary-conditional-operator), kosh == :sunglasses:

**Comment by koshalt** [comment]
lgtm

**Pull request merged**